### PR TITLE
move jjobs and exscripts to global-workflow

### PR DESCRIPTION
**Pull request by aerorahul**
This PR does what the title says. 

See the companion global-workflow PR https://github.com/NOAA-EMC/global-workflow/pull/941

### CODE REVIEW

**Review by CoryMartin-NOAA — APPROVED** [review]

File: scripts/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,5 @@
 list(APPEND scripts_files
-  exgdas_global_atmos_analysis_prep.py
-  exgdas_global_atmos_analysis_post.py
-  exgdas_global_atmos_analysis_run.sh
+  exgdas_global_marine_analysis_run.sh
```
Comment (guillaumevernieres) [inline scripts/CMakeLists.txt:2]: Oops ... Thanks.

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Review by RussTreadon-NOAA — APPROVED** [review]
Looks good to me

**Comment by CoryMartin-NOAA** [comment]
@aerorahul I will squash merge whenever you are ready to do the same in global-workflow (or I believe you have permissions to do so in this repo so just do it)

**Pull request merged**